go/trace/utils.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2017 Google Inc.
```

Comment:
I don't mind either way. This is what the existing files in the trace package had, so I just kept it. Maybe it's time to check this automatically so that all files look the same?